### PR TITLE
[main < typo-fix-20240614] Fix typos

### PR DESCRIPTION
**Pull request by kgolubic**
### Description

- Fixed typos across the repo

### Pull request type

Please check what kind of PR this is:

- [ ] Fix or improvement of an existing page

### Related PRs and issues

PR this doc page is related to: 
(especially necessary if the PR is related to a release)

Closes:
(paste the link to the issue it closes)

### Checklist:

- [ ] Check all content with Grammarly
- [ ] Perform a self-review of my code
- [ ] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/advanced-algorithms/available-algorithms/bipartite_matching.mdx [hunk 1]
```diff
@@ -83,7 +83,7 @@ MERGE (a:Node {id: 1}) MERGE (b:Node {id: 4}) CREATE (a)-[:RELATION]->(b);
 MERGE (a:Node {id: 2}) MERGE (b:Node {id: 4}) CREATE (a)-[:RELATION]->(b);
 ```
 
-{<h3> Get maximung biparte matching </h3>} 
+{<h3> Get maximum biparte matching </h3>}
```
Comment (hal-eisen-MG) [inline pages/advanced-algorithms/available-algorithms/bipartite_matching.mdx:86]: s/biparte/bipartite

File: pages/advanced-algorithms/available-algorithms/collections.mdx [hunk 2]
```diff
@@ -110,7 +110,7 @@ Unites two lists into one, eliminating duplicates.
 
 {<h4> Output: </h4>}
 
-- `List[Any]` ➡ The union of the inputed lists.
+- `List[Any]` ➡ The union of the inputted lists.
```
Comment (hal-eisen-MG) [inline pages/advanced-algorithms/available-algorithms/collections.mdx:113]: s/inputted/two input

File: pages/advanced-algorithms/available-algorithms/bridges.mdx [hunk 3]
```diff
@@ -40,7 +40,7 @@ You can execute this algorithm on [graph projections, subgraphs or portions of t
 
 ### `get()`
 
-The procedure indentifies a bridge relationships in the graph which if deleted,
+The procedure identifies a bridge relationships in the graph which if deleted,
```
Comment (hal-eisen-MG) [inline pages/advanced-algorithms/available-algorithms/bridges.mdx:43]: s/a bridge relationships/a bridge relationship
The "number" should agree. The word "a" implies a single relationship.

File: pages/advanced-algorithms/available-algorithms/create.mdx [hunk 4]
```diff
@@ -452,7 +452,7 @@ relationships. If the property already exists it is modified.
 {<h4> Input: </h4>}
 
 - `subgraph: Graph` (**OPTIONAL**) ➡ A specific subgraph, which is an [object of type Graph](/advanced-algorithms/run-algorithms#run-procedures-on-subgraph) returned by the `project()` function, on which the algorithm is run.
-- `relationships: int|Relationship|List[int|Relationship]` ➡ Relationships that need to be modified inputed using their object or ID.
+- `relationships: int|Relationship|List[int|Relationship]` ➡ Relationships that need to be modified inputted using their object or ID.
```
Comment (hal-eisen-MG) [inline pages/advanced-algorithms/available-algorithms/create.mdx:455]: Generally, "inputted" is not the best usage - "are indicated" is better here.

**Comment by kgolubic** [comment]
@hal-eisen-MG I've applied all of the suggestions. Merging to main.

**Comment by hal-eisen-MG** [comment]
> @hal-eisen-MG I've applied all of the suggestions.

Hm. I'm confused. I'm not seeing any subsequent changes. Should we jump on a quick call?

**Comment by kgolubic** [comment]
> > @hal-eisen-MG I've applied all of the suggestions.
> 
> Hm. I'm confused. I'm not seeing any subsequent changes. Should we jump on a quick call?

Sorry. I forgot to push the changes from the local branch. The changes should be on the GitHub branch as well.

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Pull request merged**